bazel-java-deps.bzl

```diff
@@ -45,50 +45,51 @@ version_specific = {
 # This effectively means all io.grpc:*, io.netty:*, and `com.google.protobuf:protobuf-java
 # need to be updated with careful consideration.
 
-netty_tcnative_version = "2.0.46.Final"
-netty_version = "4.1.72.Final"
-grpc_version = "1.44.0"
-protobuf_version = "3.19.3"
+netty_tcnative_version = "2.0.53.Final"
+netty_version = "4.1.77.Final"
+grpc_version = "1.49.2"
+protobuf_version = "3.21.7"
 akka_version = "2.6.18"
 akka_http_version = "10.2.8"
-gatling_version = "3.5.1"
-guava_version = "31.0.1-jre"
+gatling_version = "3.8.4"
+guava_version = "31.1-jre"
 
 # observability libs
-dropwizard_version = "4.1.2"
-opentelemetry_version = "1.12.0"
-prometheus_version = "0.8.1"
+dropwizard_version = "4.2.12"
+opentelemetry_version = "1.19.0"
+prometheus_version = "0.16.0"
```

Comment:
@gerolf-da We would prefer to not update the observability libraries yet. 
Adding label support for metrics currently depends on the 1.12.0 update and we're in the process of selecting an update path. But the current version allows us to easily add label support alongside dropwizard. 
Also the dropwizard library doesn't work with the current setup without some changes to our init structure. 
Already available PR to upgrade telemetry to valid versions: https://github.com/digital-asset/daml/pull/15211
Issue

Reply:
Okay, I will revert these libraries then.